### PR TITLE
add link to be clear

### PR DESCRIPTION
**Pull request by oops-oom**
Signed-off-by: oops-oom <734819342@qq.com>

to be clear with how app customize metrics

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ the example application throughout this task.
     To customize telemetry v2 metrics, for example, to add `request_host`
     and `destination_port` dimensions to the `requests_total` metric emitted by both
     gateways and sidecars in the inbound and outbound direction, change the installation
-    options as follows:
+    options with command [istioctl install -f](/docs/setup/install/istioctl/#install-istio-using-the-default-profile) and yaml as follows:
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/metrics/customize-metrics/index.md:74]: may be you can add the command in the below section along with the yaml in {{text bash}} ?

File: content/en/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 2]
```diff
@@ -109,7 +110,12 @@ the example application throughout this task.
                         request_host: request.host
     {{< /text >}}
 
-1. Apply the following annotation to all injected pods with the list of the
+
+    {{< text bash >}}
+    $ istioctl install -f custom_metrics.yaml
+    {{< /text >}}
+
+2. Apply the following annotation to all injected pods with the list of the
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/customize-metrics/index.md:118]: Normally, we would leave this as a 1. and let the markdown handle the numbering. I believe there is an issue with tabs and numbering which requires the numbering to be done manually.

File: content/en/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 3]
```diff
@@ -81,6 +81,7 @@ the example application throughout this task.
     {{< /tip >}}
 
     {{< text yaml >}}
+    # custom_metrics.yaml
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/metrics/customize-metrics/index.md:84]: I think the usual format followed is similar to https://github.com/istio/istio.io/blame/master/content/en/docs/tasks/observability/distributed-tracing/mesh-and-proxy-config/index.md#L75

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Pull request closed without merging**